tests/test_longbotcommand.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python
+#
+# A library containing community-based extension for the python-telegram-bot library
+# Copyright (C) 2020-2021
+# The ptbcontrib developers
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser Public License for more details.
+#
+# You should have received a copy of the GNU Lesser Public License
+# along with this program.  If not, see [http://www.gnu.org/licenses/].
+import subprocess
+
+import pytest
+import sys
+
+from telegram import BotCommand
+from ptbcontrib.longbotcommand import LongBotCommand
+
+
+@pytest.fixture(scope='module', autouse=True)
+def install_requirements():
+    subprocess.check_call(
+        [
+            sys.executable,
+            "-m",
+            "pip",
+            "install",
+            "-r",
+            "ptbcontrib/longbotcommand/requirements.txt",
+        ]
+    )
```

Comment:
I think this can be skipped, as this contribution has no special requirements

Reply:
Done